packages/fury-cli/lib/fury.js

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
 
+/* eslint-disable no-console */
```

Comment:
I didn't add any extra console but there are other `console.log` bringing ESLint to a warning state.